### PR TITLE
Did profile feature

### PR DESCRIPTION
**Pull request by mlopezdetrinidad**

### CODE REVIEW

**Review by rubengue — APPROVED** [review]
Great

File: pages/profile.tsx [hunk 1]
```diff
@@ -32,14 +89,38 @@ const Profile: FC = () => {
...
-      created,
-      did: 'didkeyz6kghijklXXJPT17VIakupmu89NSTYI8mni',
-      address: '0x9e81de93dC...47e6d64b70ff1dF',
-    };
-    dispatch(doSetProfile(currentProfile));
-    setProfile(currentProfile);
+    try {
+      const accountName = values.name.replaceAll(' ', '').toLowerCase();
+      const xdvWallet = new Wallet({ isWeb: true });
+      await xdvWallet.open(accountName, values.passPharse);
```
Comment (molekilla) [inline pages/profile.tsx:95]: passphrase

File: pages/profile.tsx [hunk 2]
```diff
@@ -32,14 +89,38 @@ const Profile: FC = () => {
...
+      const walletId = await xdvWallet.addWallet();
+      const walletDid = await xdvWallet.createES256K({
+        passphrase: values.passPharse,
+        rpcUrl: process.env.NEXT_PUBLIC_RPC_URL,
+        walletId,
+        registry: '',
+        accountName: values.name,
+      });
+
+      const walletSessionStorage = { walletId, profileName: values.name, createdAt: created };
```
Comment (molekilla) [inline pages/profile.tsx:110]: store the profile remotely, in ipfs/ipld

Comment (molekilla) [reply pages/profile.tsx:110]: of course, wont be available unless he does a backup

**Review by molekilla — CHANGES_REQUESTED** [review]

**Review by molekilla — APPROVED** [review]

**Pull request merged**